src/encoded/static/components/browse/components/AboveTableControls/SelectedFilesControls.js

```diff
@@ -17,18 +17,6 @@ import { wrapInAboveTablePanel } from './wrapInAboveTablePanel';
 var { Item } = typedefs;
 
 
-export class SelectedFilesOverview extends React.Component {
```

Comment:
Is not being used anywhere, likely a left-over artifact from before we had Download Buttons / Metadata TSV implemented.